CHANGES.rst

```diff
@@ -126,6 +126,9 @@ New Features
     ``Table._repr_html_``) now use consistent table styles which can be set
     using the ``astropy.table.default_notebook_table_class`` configuration
     item. [#4886]
+  - Added interface to create ``Table`` directly from any "compatible" table
+  - Added interface to create ``Table`` directly from any table-like object
```

Comment:
I don't know if this was me or rebase, but I'll fix this after tests are passing.